services/src/main/java/za/co/sfy/lucid/sams/rest/vo/data/writer/SubjectSetRequest.java

```diff
@@ -0,0 +1,53 @@
+
+package za.co.sfy.lucid.sams.rest.vo.data.writer;
+
+import java.io.Serializable;
+
+
+
+public class SubjectSetRequest implements Serializable {
+
+    private static final long serialVersionUID = 6627252016141632712L;
+
+    private Short description;
```

Comment:
May have to consider making this Integer and converting to short in your mapper. remember null check before conversion